main.py

```diff
@@ -89,12 +91,36 @@
             if time.time() - click_time < 0.3:
                 if zoomed is False:
                     background = background_zm_in
```

Comment:
this was left in so that it centers on the center of the image when done. jank solution until we figure out how to zoom from the event position